lib/repositories/firebase/participant_repository.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:pispapp/models/fsp.dart';
+import 'package:pispapp/repositories/interfaces/i_participant_repository.dart';
+import 'package:cloud_firestore/cloud_firestore.dart' show Firestore;
+
+class ParticipantRepository implements IParticipantRepository {
+  final _colRef = Firestore.instance.collection('participants');
+
+  @override
+  Stream<List<Fsp>> listenForAvailableFSPs() {
```

Comment:
Is this function still needed?

Reply:
Not really needed anymore. I can delete it before I merge